### PR TITLE
Starbtn

### PR DESCRIPTION
**Pull request by chcx97**
- modified star button
- Fixed star buttons on landing page so user can now add a dog to their starred list from  the landing page

### CODE REVIEW

File: balto-ui/src/components/StarButton/StarButton.jsx [hunk 1]
```diff
@@ -41,10 +41,12 @@ export default function StarButton({ dogId=1, dogName=""}) {
             if (isStarred) {
                 // TODO: may want to add error handling on frontend here
                 const { data, error } = await ApiClient.unstarDog(dogId)
+                console.log(data, error)
                 setIsStarred(false)
             } else {
                 // TODO: may want to add error handling on frontend here
                 const { data, error } = await ApiClient.starDog({dogId})
+                console.log(data, error)
```
Comment (LifeBringer) [inline balto-ui/src/components/StarButton/StarButton.jsx]: Remove/comment out

File: balto-ui/src/components/StarButton/StarButton.jsx [hunk 2]
```diff
@@ -41,10 +41,12 @@ export default function StarButton({ dogId=1, dogName=""}) {
             if (isStarred) {
                 // TODO: may want to add error handling on frontend here
                 const { data, error } = await ApiClient.unstarDog(dogId)
+                console.log(data, error)
```
Comment (LifeBringer) [inline balto-ui/src/components/StarButton/StarButton.jsx]: Remove/comment out

**Review by LifeBringer — CHANGES_REQUESTED** [review]
See comments

**Review by LifeBringer — APPROVED** [review]

File: balto-ui/src/components/StarButton/StarButton.jsx (hunk 1 above)
Comment (chcx97) [reply balto-ui/src/components/StarButton/StarButton.jsx]: just removed the comments

**Pull request merged**